util/cosmos/iscn/sign.js

```diff
@@ -106,8 +106,16 @@ function preformatISCNPayload(payload) {
     default: actualType = 'CreativeWork';
   }
 
-  const contentFingerprints = [];
-  if (fingerprint) contentFingerprints.push(`ipfs://${fingerprint}`);
+  let contentFingerprints = [];
+  if (fingerprints) {
+    if (Array.isArray(fingerprints)) {
+      if (fingerprints[0].startsWith('Qm') && fingerprints[0].length === 46) {
+        contentFingerprints.push(`ipfs://${fingerprints[0]}`);
+      } else {
+        contentFingerprints = contentFingerprints.concat(fingerprints);
+      }
```

Comment:
Should check each entry with for-loop instead of hardcoding `[0]`

Reply:
In fact we should handle this logic in page level, page should convert old format into new format